### PR TITLE
Feature Add and Remove Functionality with Redux

### PR DESCRIPTION
**Pull request by jubaer919**
# Highlight 🔍 

- Wrap `<App>` with the `<Provider>` component from `react-redux`.
- Pass configured `store` into the `<Provider>`.
- Create an `initialState` variable for `booksSlice`, with given `array`.
- Display  `books`, received from the slice, in a reusable component.
- Dispatch actions using `useDispatch`.
- Added the functionality for `adding` book.
- Added the functionality for `deleting` book.

### CODE REVIEW

File: src/components/books/AddBook.js [hunk 1]
```diff
@@ -1,12 +1,104 @@
...
+//   };
+
+//   const gunraChangeHandler = (event) => {
+//     setGunra(event.target.value);
+//   };
+
+//   const submitChangeHandler = (event) => {
+//     event.preventDefault();
+
+//     if (title.trim().length === 0 || author.trim().length === 0 || gunra.trim().length === 0) {
```
Comment (V-Blaze) [inline src/components/books/AddBook.js]: - You have done great so far :+1: :+1: Your app meets all the requirements and works as expected. but there are a few things you need to correct before moving to the next milestone. :smile: :wave: 

- [ ] Following the best practices, you are not meant to leave unused code as a comment on your code base. it makes your codebase look unprofessional. I'll suggest you remove all these commented-out codes . :smile: :wave:

File: src/components/books/IndividualBook.js [hunk 2]
```diff
@@ -1,46 +1,31 @@
+import React from 'react';
 // eslint-disable-next-line import/no-extraneous-dependencies
 import PropTypes from 'prop-types';
```
Comment (V-Blaze) [inline src/components/books/IndividualBook.js]: - [ ] As required by Microverse, you are not meant to use any linters disable rule in your code base. The linters are there to make sure your code follows the best industry standard. I'll suggest you remove this `disable` rule and try to fix the issue. :smile: :small_airplane:

**Review by V-Blaze — CHANGES_REQUESTED** [review]
## Hi @jubaer919 

## Required changes 🖊️ 
# Almost there 🟡 🟦 🟨 

giphy


Great work so far, You have come a long way, keep moving forward👏 👍  
You've done well implementing the project requirements, but there are still some issues I think you need to improve on. 😄 👋 


 ## Highlights
- Redux store passed to the `<Provider>` correctly✔
-  Books are loaded from redux state👋 
-  Add and remove functionality works great✔

 ## Required Changes ♻️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Ibizugbe — APPROVED** [review]
Hi @jubaer919 👋🏼👋🏼👋🏼, 

approved


Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights✅✅✅

- [x] Commented block of code has been removed as requested by the previous reviewer👍🏼.
- [x] `eslint-disable` has been removed as required👍🏼.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**